examples/room/index.html

```diff
@@ -15,7 +15,7 @@ <h1 class="heading">Room example</h1>
       </p>
       <div class="room">
         <div>
-          <video id="js-local-stream"></video>
+          <video id="js-local-stream" playsinline></video>
```

Comment:
Prefer https://github.com/skyway/skyway-js-sdk/pull/181/files#diff-f1febb3bb1ec01a6a77d8329ed4a2cafR21

Reply:
Thank you. I'll fix it.